app/views/users/index.html.erb

```diff
@@ -28,7 +28,7 @@
       <%= link_to 'Delete', user_post_path(@user.id, post.id), method: "delete" %>
       <%= link_to 'Update', user_post_path(@user.id, post.id), method: "get" %>
     <% end %>
-    <p>Likes: <%= post.likes.length %></p>
+    <p><%= link_to 'Likes:', user_post_likes_path(@user.id, post.id) %> <%= post.likes.length %></p>
```

Comment:
Can we please change Likes: to Likes
ie. remove semi-colon

Reply:
Is it difficult to add this to the user profile page as well? If it's more than a 15 min job don't worry - but would be cool to have it there as well.